dbt_airflow_factory/k8s/k8s_operator.py

```diff
@@ -52,10 +52,15 @@ def create(
         return self._create(self._prepare_arguments(command, model, additional_dbt_args), name)
 
     def _prepare_arguments(
-        self, command: str, model: Optional[str], additional_dbt_args: Optional[List[str]]
+        self,
+        command: str,
+        model: Optional[str],
+        additional_dbt_args: Optional[List[str]],
     ) -> List[str]:
         args = [
-            "set -e;",
+            "./cmd_executor.sh"
+            if self.kubernetes_execution_parameters.execution_script
+            else "set -e;",
             f"dbt --no-write-json {command}",
```

Comment:
we dicussed this "dbt --no-write-json" should go into else

Reply:
Changed: 86f643c19307ffd429c599177f3396e4eb186920